var/spack/repos/builtin/packages/py-sphinxext-opengraph/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PySphinxextOpengraph(PythonPackage):
+    """Sphinx Extension to enable OGP support"""
+
+    homepage = "https://github.com/wpilibsuite/sphinxext-opengraph"
+    pypi = "sphinxext-opengraph/sphinxext-opengraph-0.6.3.tar.gz"
+
+    maintainers = ["JBlaschke"]
+
+    version("0.6.3", sha256="cd89e13cc7a44739f81b64ee57c1c20ef0c05dda5d1d8201d31ec2f34e4c29db")
+
+    depends_on("py-setuptools", type="build")
```

Comment:
Missing deps:
```
    install_requires=["sphinx>=2.0"],                                                    
    python_requires=">=3.6",                                                             
```

Reply:
@JBlaschke are you still working on this?

Reply:
@bernhardkaindl let me try and find some time over the next week to finish this ...